scripts/tests/implement_issue.Tests.ps1

```
@@ -1,34 +1,34 @@
 # git reset --hard, git push, gh pr create) and no extracted function to call
 # in isolation, so these tests extract the discovery snippet verbatim from the
 # source file's text and dot-source it with git/Test-Path mocked. This runs the
 # exact shipped code (not a re-implementation), so a regression in the real
 # script fails these tests, while `git`/the filesystem are never touched.
 #
-# Constraint: this file must never modify aider-issue.ps1 (tests only).
+# Constraint: this file must never modify implement_issue.ps1 (tests only).
 
 BeforeAll {
-    $script:ScriptUnderTest = (Resolve-Path (Join-Path $PSScriptRoot '..' 'aider-issue.ps1')).Path
+    $script:ScriptUnderTest = (Resolve-Path (Join-Path $PSScriptRoot '..' 'developer_tools' 'implement_issue.ps1')).Path
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Join-Path' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)